src/types/internal.ts

```diff
@@ -47,6 +47,7 @@ export type DataFormat = "raw" | "bufferLayout" | "borsh";
 export interface IInstructionData {
   format: DataFormat;
   raw: string;
+  isHex: boolean;
```

Comment:
Let's have `raw: { content: string; encoding: "hex" | "bs58" }` instead. 

Please note that should also update `IInstructionExt` and the relevant mappers

Reply:
* add an interface IRaw
* use bs58 encoded raw data in mapITransactionToTransactionExt